src/throttler-storage-redis.service.ts

```diff
@@ -25,21 +22,49 @@ export class ThrottlerStorageRedisService implements ThrottlerStorageRedis, OnMo
     }
   }
 
+  /**
+   * @description: This method look for members (records) in a Redis set, see: https://redis.io/docs/data-types/sets/
+   *
+   * @param key
```

Comment:
If you're adding this, then make sure to follow [jsdoc @param](https://jsdoc.app/tags-param.html) documentation and provide a small description of this key.

Reply:
Removed, those annotations are useless for now